stable/datadog/templates/daemonset.yaml

```diff
@@ -0,0 +1,56 @@
+{{- if .Values.datadog.apiKey }}
+apiVersion: extensions/v1beta1
+kind: DaemonSet
```

Comment:
I think it would make sense to enable the Kubernetes integration by default on this Helm chart. Probably would make sense to load that in via configMap